src/main/java/com/google/swarm/tokenization/DLPTextToBigQueryStreamingV2.java

```diff
@@ -101,13 +107,42 @@ public static PipelineResult run(DLPTextToBigQueryStreamingV2PipelineOptions opt
 
   private static void runInspectAndDeidPipeline(
       Pipeline p, DLPTextToBigQueryStreamingV2PipelineOptions options) {
-    PCollection<KV<String, ReadableFile>> inputFiles =
-        p.apply(
-                FilePollingTransform.newBuilder()
-                    .setFilePattern(options.getFilePattern())
-                    .setInterval(DEFAULT_POLL_INTERVAL)
-                    .build())
-            .apply("Fixed Window", Window.into(FixedWindows.of(WINDOW_INTERVAL)));
+    PCollection<KV<String, ReadableFile>> inputFiles;
+
+    if (options.getInputProviderType() == InputLocation.GCS && options.getGCSNotificationTopic() == null
+        && !options.getProcessExistingFiles())
+        throw new IllegalArgumentException("Either --processExistingFiles or --gcsNotificationTopic should be provided");
+
```

Comment:
Please always use
if() {
    // even for single statement. Improves code readability significantly
} else {
   // even for single statement. Improves code readability significantly
}